src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,81 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { getMovieByTitle } from '../../api/api';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  onAdd: (movie?: Movie) => void;
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [query, setQuery] = useState<string>('');
+  const [movie, setMovie] = useState<Movie>();
+  const [warn, setWarn] = useState<boolean>(false);
+
+  async function getMovie() {
+    const response = await getMovieByTitle(query);
+
+    if (response.Response === 'True') {
+      setMovie(response);
+    } else {
+      setWarn(true);
+    }
+  }
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              value={query}
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
+              onChange={(e) => setQuery(e.target.value)}
```

Comment:
Мені вже набридло це писати в окремих пулах. У нас не існує майже однобуквенних змінних. `e` - це може бути event, error і тд. Краще явно чим не явно
```suggestion
              onChange={(event) => setQuery(event.target.value)}
```